common/templates/common/blocks/image.html

```diff
@@ -1,4 +1,5 @@
 {% load wagtailcore_tags %}
 {% load wagtailimages_tags %}
+
 {% image value original as value %}
-<img src='{{ value.url }}' alt='{{ value.alt }}' />
+<img style="height: auto; display: block; margin: auto auto;" src='{{ value.url }}' alt='{{ value.alt }}' />
```

Comment:
I'd like to have styling in .css files. Is there a way we can make this a class rather than inline?